tools/crm_node.c

```diff
@@ -721,7 +716,14 @@ remove_node(const char *target_uname)
         }
     }
 
-    exit_code = CRM_EX_OK;
+    // Lastly, purge the node from the CIB itself
+    rc = purge_node_from_cib(node_name, nodeid);
+    if (rc != pcmk_rc_ok) {
+        exit_code = pcmk_rc2exitc(rc);
+        g_set_error(&error, PCMK__EXITC_ERROR, exit_code,
+                    "Could not purge node %s from CIB " XML_CIB_TAG_STATUS
+                    ": %s", pcmk__s(node_name, "by ID"), pcmk_rc_str(rc));
```

Comment:
Here's a crazy future idea... pcmk__s could be modified to have its second argument be a format string.  Then you could do stuff like `pcmk__s(node_name, "by ID %d", nodeid)`.  This wouldn't require changes to existing callers.

Reply:
That would be nice, but it would require it to be dynamically allocated and freed, and I think being inline is its main advantage

Reply:
`pcmk__s()` could use a global freed in `crm_exit()` but that seems overkill